packages/template-block/src/hooks/useTemplateBlockData.ts

```diff
@@ -0,0 +1,147 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import {
+    Template,
+    TemplateLegacy,
+    closeTemplateChooser,
+    useBlockAssets,
+    useBlockSettings,
+    useBlockTemplates,
+    useEditorState,
+} from '@frontify/app-bridge';
+import { TextStyles, generateRandomId } from '@frontify/fondue';
+import { useCallback, useEffect, useState } from 'react';
+import { PreviewType, Settings, TextPositioningType, textPositioningToStyles } from '../types';
+import { TEMPLATE_BLOCK_SETTING_ID } from '../constants';
```

Comment:
Please re-order imports to match web-app. Although we don't have eslint enforced here it adds to readability.

Also any type imports should be explicitly set as type (eg: import { type ReactElement } from 'react')